.buildkite/pr.yml

```diff
@@ -0,0 +1,47 @@
+tapendium-aliases:
+  lib-common: &lib-common
+    - seek-oss/github-merged-pr#v1.1.2:
+        mode: checkout
+    - tapendium/1password-secrets#v2.3.0:
+        file:
+          - path: .npmrc.tpl
+            out: .npmrc
+    - nienbo/cache#v2.4.15:
```

Comment:
**suggestion:** Add the `pipeline slug override` option in the PR and merge pipelines to reuse the npm cache between both.
e.g. https://github.com/tapendium/template-email/blob/main/.buildkite/pr.yml#L22

Reply:
yes! 6102fbc